geometric_features/aggregation/ocean/arctic_regions.py

```diff
@@ -0,0 +1,22 @@
+def arctic(gf):
+    """
+    Aggregate ocean-relevant Arctic regions
+
+    Parameters
+    ----------
+    gf : geometric_features.GeometricFeatures
+        An object that knows how to download and read geometric features
+
+    Returns
+    -------
+    fc : geometric_features.FeatureCollection
+        The new feature collection with ocean-relevant Arctic regions
+    """
+    # Authors
+    # -------
+    # Milena Veneziani, Xylar Asay-Davis
+
+    fc = gf.read(componentName='ocean', objectType='region',
+                 tags=['Arctic'])
```

Comment:
so basically here we would add the `Arctic_Proshutinsky` tag, and then add a few lines of code to aggregate the regions to make Arctic-noBarentsKaraSeas. I can do this if you like.